### PR TITLE
added geeksrus-mqtt-panel V1.0.7

### PR DESCRIPTION
**Pull request by geeks-r-us**

### CODE REVIEW

**Comment by geeks-r-us** [comment]
Any progress on adding this to the repo?

**Comment by marcusolsson** [comment]
Thank you for considering contributing your plugin! 

It looks like the commit is missing the `dist` directory, which is needed for publishing. For more details on publishing a plugin, check out [Publish your plugin](https://grafana.com/tutorials/build-a-panel-plugin/#8).

**Comment by marcusolsson** [comment]
Could you please take a look at the issues reported by [the plugin validator](https://grafana-plugins-web-vgmmyppaka-lz.a.run.app/) (enter `geeks-r-us/mqtt-panel@aafa02c4c646fba28ae4daaaed2415019e71709f` in the text field and press Enter).

File: repo.json [hunk 1]
```diff
@@ -3862,6 +3862,18 @@
           "url": "https://github.com/grafana/strava-datasource"
         }
       ]
+    },
+    {
+      "id": "geeksrus-mqtt-panel",
+      "type": "panel",
+      "url": "https://https://github.com/geeks-r-us/mqtt-panel",
```
Comment (marcusolsson) [inline repo.json:3869]:
```suggestion
      "url": "https://github.com/geeks-r-us/mqtt-panel",
```

**Comment by marcusolsson** [comment]
Thank you for your patience while waiting for us to review your plugin. 

We're working hard at improving the plugin experience for both users and authors. I'd like to share some important information on how to package and publish plugins going forward.

Since Grafana 7.3, we require all plugins to be _signed_. By signing your plugin, users can guarantee the integrity of the plugin and that the plugin version they've installed hasn't been modified since it was released. For more information on how to sign your plugin, refer to [Sign a plugin](https://grafana.com/docs/grafana/latest/developers/plugins/sign-a-plugin/).

With this, we now require plugins to be _packaged_ before we publish them. This means that instead of a commit containing a `dist` directory, you will need to submit a link to a ZIP archive containing your plugin. For more information on how to package your plugin, refer to [Package a plugin](https://grafana.com/docs/grafana/latest/developers/plugins/package-a-plugin/).

Once you've packaged your plugin, you'll need to update your entry to `repo.json` to the new format. You can read more on how to do this under [Add a plugin to repo.json](https://github.com/grafana/grafana-plugin-repository#add-a-plugin-to-repojson).

To summarize, to publish your plugin, make sure that you:

1. [Sign your plugin](https://grafana.com/docs/grafana/latest/developers/plugins/sign-a-plugin/)
2. [Package your plugin](https://grafana.com/docs/grafana/latest/developers/plugins/package-a-plugin/)
3. Create a new GitHub release and upload the packaged plugin as a release asset
4. [Update your repo.json entry](https://github.com/grafana/grafana-plugin-repository#add-a-plugin-to-repojson)

Here's an [example](https://github.com/marcusolsson/grafana-jsonapi-datasource/releases/tag/v0.6.0) of a properly packaged plugin. In the same repository, you can find a [GitHub Action](https://github.com/marcusolsson/grafana-jsonapi-datasource/blob/master/.github/workflows/main.yml) that signs, packages, and releases the plugin whenever you create a new tag.

Once you've created a GitHub release with the packaged plugin, you can right-click the release asset, copy the link, and validate it using [the plugin validator](https://grafana-plugins-web-vgmmyppaka-lz.a.run.app). 

Finally, in the November [community call](https://groups.google.com/g/grafana-developers/c/ps2vzCXRDCM/m/i7uAK1u9AQAJ), I'll be walking through the steps needed to sign and package your plugin. A recording will be made available on our [YouTube account](https://www.youtube.com/playlist?list=PLDGkOdUX1Ujo0OyE5HFffHE9GJVvaVXGs), in case you can't make it.

**Comment by geeks-r-us** [comment]
Changed to new zip file version

**Comment by marcusolsson** [comment]
There seems to be some issues with the pull request. I don't see any changes to the repo.json, only changes to other files in the repository. Could you take a look at it?

**Comment by geeks-r-us** [comment]
the changed files are due to rebase didn't work.
The actual change is in a2eb423

**Comment by marcusolsson** [comment]
Hm. It might be easier if you create a new PR altogether?

**Comment by geeks-r-us** [comment]
New one is #866

**Pull request closed without merging**